### PR TITLE
mgrpxy: expose port on IPv6 network (bsc#1227951)

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

followup of https://github.com/uyuni-project/uyuni-tools/pull/406. Same changes should be present in `mgrpxy`.

## Test coverage
- No tests
- [x] **DONE**

## Links

Issue(s): https://github.com/SUSE/spacewalk/issues/24848

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: mgrpxy/shared/templates/pod.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ ExecStartPre=/bin/sh -c '/usr/bin/podman pod create --infra-conmon-pidfile %t/uy
 		--network {{ .Network }} \
         {{- range .Ports }}
         -p {{ .Exposed }}:{{ .Port }}{{ if .Protocol }}/{{ .Protocol }}{{ end }} \
+	-p [::]:{{ .Exposed }}:{{ .Port }}{{if .Protocol}}/{{ .Protocol }}{{end}} \
```
Comment (vzhestkov) [inline mgrpxy/shared/templates/pod.go:36]: I think `[::]` is making it listening for IPv6 and IPv4 as well (at least it works this way with `salt-broker` directly), so 2 entries could be redundant here, but as @aaannz mentioned maybe it makes sense to check if IPv6 is enabled in the network as it's made in `mgradm` case, but I'm not fully sure here as I'm not deep familiar with `podman` network internals.

**Comment by mbussolotto** [comment]
@aaannz @vzhestkov yes, this would work also if ipv6 is disabled. I followed what we implemented for server and trying to keep it simplest as possible, but enabling it just when `ipv6` is enabled is not complicated. I'll go with this change

**Comment by vzhestkov** [comment]
Hmm, it's really interesting. With `salt-broker` if it's set to listen to `[::]` it makes it responsive for both IPv6 and IPv4, but in case of `conmon` with podman it looks like we need to have it listening to both `[::]` and `0.0.0.0`.

**Review by vzhestkov — APPROVED** [review]
:+1:

**Review by cbosdo — APPROVED** [review]

**Pull request merged**